### PR TITLE
Add link to 1.21.x documentation

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
Test this after merging #5932

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ If you are a researcher, a (bio)informatician, a biomedical practitioner, or any
 
 ### Documentation for previous versions
 
-[MOLGENIS v1.21.x](link TODO)
+[MOLGENIS v1.21.x](https://molgenis.gitbooks.io/molgenis/content/v/1.21/)
```
Comment (fdlk) [inline docs/README.md]: Can the link be relative?

Comment (Mark-de-Haan) [inline docs/README.md]: No because you are pointing to the content of another branch

Comment (fdlk) [inline docs/README.md]: Eeew. Let's use https://plugins.gitbook.com/plugin/versions instead then?

Comment (Mark-de-Haan) [inline docs/README.md]: URL will never change, why fancy if can be easy and straightforward?

Comment (fdlk) [inline docs/README.md]: User experience is better
* clear on each documentation page what version of the documentation you're browsing
* ability to go back and forward between versions

Developer experience is better
* (once set up) is easier to maintain cause all versions show same list from single source of truth

I've put up pull request to use it, can you take a look?

**Pull request closed without merging**